README.md

```diff
@@ -143,6 +143,24 @@ Modify the extension's settings in your `settings.json`.
 }
 ```
 
+### Kate with [LSP Client Plugin]
+
+[LSP Client Plugin]: https://docs.kde.org/stable5/en/kate/kate/kate-application-plugin-lspclient.html
+
+Add this to your "User Server Settings" in LSP Client configuration:
+
+```json
+{
+  "servers": {
+    "nix": {
+      "command": ["nil"],
+      "url": "https://github.com/oxalica/nil",
+      "highlightingModeRegex": "^Nix$"
```

Comment:
Could you also add an example to pass LSP settings like others?
See also https://github.com/oxalica/nil/pull/37#issuecomment-1328655292 ?

Reply:
The problem is that the others use JSON commenting syntax, which I'm not sure whether Kate parses. So if I add "comments" in the example, I'm not sure if it will still work with a straight copy-paste.